packages/sw-cache-expiration/src/lib/cache-expiration.js

```diff
@@ -193,17 +200,34 @@ class CacheExpiration {
    * Expires entries, both based on the the maximum age and the maximum number
    * of entries, depending on how this instance is configured.
    *
+   * A mutex is used to ensure that there is only one copy of this asynchronous
```

Comment:
Thanks for expanding beyond what was noted in the comment. Your notes on scheduling make sense.